cockroachdb/manifests-cert-manager/scripts.yaml

```diff
@@ -19,12 +19,16 @@ data:
 
     SQL_CMD="/cockroach/cockroach -d system sql" > /dev/null
 
+    # try creating user, database, grant database permissions to user
     $SQL_CMD << EOF
       CREATE USER IF NOT EXISTS $DB_USER;
       CREATE DATABASE IF NOT EXISTS $DB_NAME;
       GRANT ALL ON DATABASE $DB_NAME TO $DB_USER;
-      GRANT ALL ON TABLE $DB_NAME.* TO $DB_USER;
     EOF
+    
+    # If database already exists and contains tables, grant access to them for user. 
+    # This command will fail for freshly initialized databases.
+    $SQL_CMD --execute "GRANT ALL ON TABLE $DB_NAME.* TO $DB_USER;" || true
```

Comment:
I don't suppose there's an easy way to make this conditional in the SQL like `TABLES_COUNT > 0 && GRANT ... ` so we avoid any confusing/worrying looking logs in the case no tables exist

Reply:
[Cockroachdb doesn't have conditional statements, it seems](https://github.com/cockroachdb/cockroach/issues/25093).
That's why I came up with this solution. Currently, logs are looking like

```
Cluster is not ready yet. Retry 1/20. Waiting for 3 seconds...
Cluster is not ready yet. Retry 2/20. Waiting for 3 seconds...
Cluster is not ready yet. Retry 3/20. Waiting for 3 seconds...
Cluster is not ready yet. Retry 4/20. Waiting for 3 seconds...
Cluster is not ready yet. Retry 5/20. Waiting for 3 seconds...
Cluster is not ready yet. Retry 6/20. Waiting for 3 seconds...
CREATE ROLE
CREATE DATABASE
GRANT
ERROR: no object matched
SQLSTATE: 42704
Failed running "sql"
```

Which is not awful, I think. 
But why not, [I made them nicer here](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/50/commits/0902cde7919ae00843a7b2fac7d466aa1ee9887c)